src/router.js

```diff
@@ -84,6 +85,9 @@ const router = new Router({
         {
           path: 'account', alias: '', name: 'user', component: TabPersonalDetails,
         },
+        {
+          path: 'changepassword', alias: '', name: 'changepassword', component: TabChangePassword,
```

Comment:
Remove the `alias`. The `alias` is only for the default path. I'm not sure what happens when you add multiple aliases with the same path, but nothing good for sure.